Cargo.toml

```diff
@@ -20,7 +20,8 @@ base64 = "0.9.0"
 byteorder = "1.1.0"
 bytes = "0.4.6"
 httparse = "1.2.3"
-input_buffer = "0.1.1"
+# Required until input_buffer has new version published with Debug derive fix present
+input_buffer = { git = "https://github.com/snapview/input_buffer" }
```

Comment:
Update back to crate-io version after new version published.
EX: `input_buffer = "0.1.2"`